GoogleGram/app/src/main/java/com/example/zghadyali/googlegram/FeedFragment.java

```diff
@@ -0,0 +1,122 @@
+package com.example.zghadyali.googlegram;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.webkit.WebView;
+import android.widget.Button;
+
+import java.util.ArrayList;
+
+/**
+ * The FeedFragment reads any of the links stored in the SQLite database on the user's phone and
+ * loads them for the user to see and scroll through. The user can delete images from their feed and
+ * can switch back to the searchGoogle fragment to search for more images
+ */
+public class FeedFragment extends Fragment {
+
+    public Button switchGoogle;
+    public Button nextImage;
+    public Button prevImage;
+    public Button delete;
+    public ArrayList<String> imagesLink = new ArrayList<String>();
+    public WebView web;
+    public int image_index;
+    public FeedReaderDbHelper DbHelper;
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        //initializes the DbHelper to read from a SQLite database on the user's phone and load the
+        //saved images in the feed
+        DbHelper = new FeedReaderDbHelper(getContext());
+        //initializes the WebView and read the links stored in the database and returns them as an
+        //ArrayList of strings and loads the first link in the list. If the ArrayList has nothing in
+        // it, loads an empty WebView
+        View rootView = inflater.inflate(R.layout.feed, container, false);
+        web = (WebView) rootView.findViewById(R.id.rendered_image);
+        imagesLink = DbHelper.readItemsfromDB();
+        image_index = 0;
+        if (imagesLink.size() > 0) {
+            web.loadUrl(imagesLink.get(image_index));
+        }
+        else{
+            web.loadUrl("about:blank");
```

Comment:
Can you put a local placeholder image here instead? Something like; "No images in your feed!" instead of just a blank page